src/nvim/event/multiqueue.c

```diff
@@ -140,6 +140,16 @@ void multiqueue_put_event(MultiQueue *self, Event event)
   }
 }
 
+/// Move events from src to dest.
+void multiqueue_move_events(MultiQueue *dest, MultiQueue *src)
```

Comment:
I ordered the arguments the same way as `memcpy`, `memmove`, etc. If anyone feels strongly about reversing the order let me know (putting `src` first).